build.sh

```diff
@@ -90,7 +90,7 @@ fi
 # FIXME: not sure if we really need this. even if we do, it should be moved to mruby/Rakefile
 if [ -d "./mruby/${BUILD_DIR}" ]; then
     echo "mruby Cleaning ..."
-    (cd mruby && ./minirake clean)
+    (cd mruby && rake clean MRUBY_CONFIG=../build_config.rb)
```

Comment:
I'm wondering if you can use "make clean_mruby" with the following modification instead of "(cd mruby && rake clean MRUBY_CONFIG=../build_config.rb)". 

~~~
diff --git a/Makefile.in b/Makefile.in
index 2d78241..564a8e7 100644
--- a/Makefile.in
+++ b/Makefile.in
@@ -70,7 +70,7 @@ build_mruby: $(MRUBY_BUILD_DEPEND_TARGETS)
        cd $(MRUBY_ROOT) && $(RAKE) MRUBY_CONFIG=$(NGX_MRUBY_ROOT)/build_config.rb NGX_MRUBY_CFLAGS="$(NGX_MRUBY_CFLAGS)"
 
 clean_mruby:
-       cd $(MRUBY_ROOT) && $(RAKE) deep_clean && rm -rf build
+       cd $(MRUBY_ROOT) && $(RAKE) MRUBY_CONFIG=$(NGX_MRUBY_ROOT)/build_config.rb deep_clean && rm -rf build
 
 $(MRUBY_MAK_FILE): build_mruby
~~~

I don't test it by myself. If it works to you, we can remove the above FIXME comment too.

Reply:
I see. I will try it your way later and will delete above comment with different pull request. Thanks.